test/publish.test.js

```diff
@@ -49,24 +50,18 @@ describe('Test publish plugin', () => {
     nock('https://admin.hlx.page')
       .post('/live/adobe/blog/main/en/topics/bla')
       .reply(200);
-    nock('https://main--blog--adobe.hlx.live')
-      .get('/en/topics/bla')
-      .reply(200, 'bla');
-    nock('https://blog.adobe.com')
-      .get('/en/topics/bla')
-      .reply(200, 'bla');
     const { requestsMade, navigated } = await new SidekickTest({
       browser,
       page,
       plugin: 'publish',
-      waitNavigation: 'https://blog.adobe.com/en/topics/bla',
+      waitNavigation: 'https://blog.adobe.com/en/topics/bla?nocache=',
     }).run();
     const publishReq = requestsMade.find((r) => r.method === 'POST');
     assert.ok(
       publishReq && publishReq.url.startsWith('https://admin.hlx.page/live/'),
       'Live API not called',
     );
-    assert.strictEqual(navigated, 'https://blog.adobe.com/en/topics/bla', 'Redirect not sent');
+    expect(navigated.startsWith('https://blog.adobe.com/en/topics/bla?nocache=')).to.equal(true);
```

Comment:
You could have also used:
```suggestion
    assert.ok(navigated.startsWith('https://blog.adobe.com/en/topics/bla?nocache='), 'Redirect not sent');
```